samples/system-test/buckets.test.js

```diff
@@ -112,6 +117,33 @@ it("should disable a bucket's uniform bucket-level access", async () => {
   );
 });
 
+it("should change a bucket's default storage class", async () => {
+  const output = execSync(
+    `node changeDefaultStorageClass.js ${bucketName} COLDLINE`
+  );
+  assert.match(
```

Comment:
Instead of dealing with the regex, you can `assert.include(output, 'The text you want to check for')`, just to simplify things a little :)